blocks/api/registration.js

```diff
@@ -158,6 +158,15 @@ export function registerBlockType( name, settings ) {
 		set( settings, [ 'supports', 'inserter' ], ! settings.isPrivate );
 	}
 
+	if ( 'useOnce' in settings ) {
+		deprecated( 'useOnce', {
```

Comment:
I think we should make this message more detailed like:

`useOnce property in the settings param passed to wp.block.registerBlockType`

Here and in https://github.com/WordPress/gutenberg/pull/7293/files#diff-07b8f76ae05a372b6af475be908fcef2R3.